paper/paper.bib

```diff
@@ -184,3 +184,10 @@ @inproceedings{belak2019exaam
   pages={C22--010},
   year={2019}
 }
+
+@misc{all,
+    Author = {R.~Halver and S.~Schulz and G.~Sutmann},
+    Howpublished = {{\verb+https://gitlab.version.fz-juelich.de/SLMS/loadbalancing/-/releases+}},
+    Title = {{ALL - A loadbalancing library, C++ / Fortran library}},
+    Url = {https://gitlab.version.fz-juelich.de/SLMS/loadbalancing/-/releases},
```

Comment:
We would rather link to the webpage itself (http://slms.pages.jsc.fz-juelich.de/websites/all-website/) than the GitLab one. Maybe just change URL to the webpage and leave howpublished alone to link to both.

Reply:
Done